### PR TITLE
Added documentation for SSO configurations

### PR DESCRIPTION
**Pull request by koolkicks88**

### CODE REVIEW

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+# Managing SSO Configurations
+
+## Required Privileges
+
+In order to access Server Options page and configure the Single Sign-On setting, you must have the following:
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: Add "the" before "Server Options"

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+- **Role**: User must be assigned to the Role_ocadm.
+
+---
+
+## Configuring SSO Settings
+
+To configure SSO Setting, go to **Library** > **Server Options** > click on the **SSO Setting** panel.
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: Setting --> Settings
and 
SSO Setting --> SSO

Comment (heatherRose) [reply docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: It should also read as SSO in our UI (drop "settings" in the UI if it's currently there).  The word "setting" is redundant on a configuration screen.  We don't want to see tabs for SSO settings, XYZ settings, ABC settings etc...

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+#### SSO Configuration Fields
+
+![Server Options - SSO Settings](../../../../../Resources/Images/SM/Library/ServerOptions/sso-settings-overall.png "SSO Settings Panel")
+
+1. Click on the **Switch** to enable access to the SSO login button.
+
+1. Enter a **Provider**: the identity provider (IdP) for the client. Options include **Okta**, **Azure**, and **Other**.
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: Use a "." instead of a ":" and maybe reword to something like this: 

1. Enter a **Provider**.  The identity provider (IdP) for the client should be selected from the **Provider** dropdown. Options include **Okta**, **Azure**, and **Other**.

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+1. Enter a **Authority**: the URL of the OIDC/OAuth2 provider.
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: Use a "." instead of a ":" and maybe reword to something like this: 

1. Enter a **Authority**.  **Authority** is the URL of the OIDC/OAuth2 provider.

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+
+1. Enter a **Client ID**: your client application's identifier as registered with the OIDC/OAuth2.
+
+1. Enter a **Hostname**: the hostname property of the URL interface is a string containing the domain name of the URL (include the port number if applicable). For example, https://**hostname:443**.com.
+
+1. Enter an **Audience**: this identifies the recipients that the JWT is intended for.
+
+1. Enter an **Scope**: the scopes being requested from the OIDC/OAuth2 provider (default: "openid").
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: Same formatting suggestions for lines 25-39 as noted above.

Reason being is a ":" typically indicates a list of things.  Here we are writing to them in complete sentences.

I like your pattern of "tell them what to do" and then "tell them where to find the corresponding value for entry into that field."

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+
+1. Enter a **Group Name**: this should match the group name used in your IdP.
+
+1. Enter a **Role**: this value represents a valid **OpCon** role.
+
+1. Select **Save**.
+
+:::note
+
+1.  If the switch is the **On** position, the user must input values in all fields including values for **Group Mappings**.
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: If the switch is [in] the

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+2.  Examples of how to gather these requirements will be posted below (for [**Okta Application**](#okta-application), and [**Azure Application**](#azure-application)).
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: Please put the word "the" in front of both "Okta Application" and "Azure Application".
No comma before the "and"

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+    - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-application-start.png "Okta - Dashboard")
+1.  Click the "Create App Integration" button
+    - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-app-integration.png "Okta - Applications")
+    1.  Inside the modal, select the following options: 1. Sign-in method: OIDC - OpenID Connect 1. Application type: Single-Page Application
+    - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-app-integration.png "Okta - Integration Options")
+    1. Click on "Next"
+    1. Set the following options:
+       1. Grant type: Authorization
+       1. Sign-in redirects URIs:(sample) `https://<hostaname:443>/login/callback`
+          1. The **hostname** will be used in the SSO Configuration page in Solution Manager. Make sure to include the **port number** if applicable
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: 1. The **hostname** will be used [on] the SSO Configuration page in Solution Manager. Make sure to include the **port number** if applicable

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+       1. The value for URL to access Solution Manager will be used in the configuration settings
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: Please add "the" before URL

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+    1. Click on "Add Claim"
+       1. The following is an example on how to add a custom claim
+       - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-group-claim.png "Okta - Add Group Claim")
+       1. Make sure the token type is Access Token and that you enter **`.*`** for the regex logic
+       1. Make sure to name claims “**groups**”
+       1. Repeat the process to add an “**email**” claim
+          1. Make sure to add **appuser.email** in the **Value** text field
+       - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-claim-email.png "Okta - Add Email Claim")
+    1. Click on the “Settings” tab and take notes of the following values:
+       1. Audience will be used in the configuration settings page inside Solution Manager
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: 1. Audience, this will be used in the configuration settings page inside Solution Manager

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+       1. Issuer will be used as the “Authority” value inside Solution Manager
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: 1. Issuer, this will be used as the “Authority” value inside Solution Manager

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+       - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-server-settings.png "Okta - Default Server Settings")
+    1. Go to the "Token Preview" tab
+       1. Select your new created application
```
Comment (heatherRose) [inline docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: 1. Select your newly created application

Also, did something make all the bullet points into "1." or is that just some weird rendering in github?

**Review by heatherRose — CHANGES_REQUESTED** [review]

File: docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md (hunk 12 above)
Comment (koolkicks88) [reply docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: The markup we use in Docasaurus automatically numbers them so I don't have to keep track 😄

Comment (heatherRose) [reply docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md]: Cool.  Just checking.

**Review by heatherRose — APPROVED** [review]

**Review by heatherRose — DISMISSED** [review]

**Review by rweesner — DISMISSED** [review]

**Comment by rweesner** [comment]
Hi all, I approved the initial PR but will hold off on pushing the changes until all of your sub conversations are resolved with potential changes. Let me know when you've completed.

**Review by rweesner — APPROVED** [review]
